components/micro-gateway-core/src/main/ballerina/gateway/filters/subscription_filter.bal

```diff
@@ -30,17 +30,17 @@ public type SubscriptionFilter object {
     public function filterRequest (http:Listener listener, http:Request request, http:FilterContext filterContext)
                         returns boolean {
         string authScheme = runtime:getInvocationContext().authContext.scheme;
-        log:printDebug("Auth scheme: " + authScheme);
+        printDebug(KEY_SUBSCRIPTION_FILTER, "Auth scheme: " + authScheme);
         if(authScheme == AUTH_SCHEME_JWT ){
             string jwtToken = runtime:getInvocationContext().authContext.authToken;
             string currentAPIContext = getContext(filterContext);
             AuthenticationContext authenticationContext;
             match getEncodedJWTPayload(jwtToken) {
                 string  jwtPayload => {
-                    log:printTrace("Encoded JWT payload: " + jwtPayload);
+                    printTrace(KEY_SUBSCRIPTION_FILTER, "Encoded JWT payload: " + jwtPayload);
```

Comment:
Since this is a self contained token, isn't it a security issue to print the token

Reply:
Hi @Rajith90 this is only the payload part of the token. This can't be used to invoke the API.